tensorflow/compiler/xla/service/sort_simplifier.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2019

Reply:
@sanjoy should it be 2018 or 2017?

Reply:
Oops, made the comment on the wrong file; I meant on `sort_simplifier_test.cc`.

Reply:
Thanks, done now :)